odoo/addons/base/views/ir_model_views.xml

```diff
@@ -323,7 +323,7 @@
                                 </group>
                                 <group string="Advanced Properties" groups="base.group_no_one">
                                     <field name="related"/>
-                                    <field name="depends" required="compute not in [False, '']"/>
+                                    <field name="depends" required="compute not in [False, ''] and store == True"/>
```

Comment:
Couldn't we simplify the condition?  This one was automatically converted from a domain.
```suggestion
                                    <field name="depends" required="bool(compute) and store"/>
```